src/GraphQL/VariableName.cs

```diff
@@ -1,14 +1,26 @@
 namespace GraphQL
 {
+    /// <summary>
+    /// Represents the full path to a property of a variable, typically for
```

Comment:
>to a _property_ of a variable

Sounds strange to me.

Reply:
Yeah, anything you suggest is fine with me.  That was the best I could come up with at the time - lol

Reply:
> Represents the full path to a variable

?

Reply:
The class might represent "variableA.products[4].categoryName".  In this case the name is now "variableA.products" and the index is 4 and the ChildName is "categoryName".  Somehow the XML comment needs to explain this concept to the user, that the class as a whole represents, as I called it, "the full path to a property of a variable".  I'm open to suggestions.  But it certainly isn't just a variable, and isn't just a variable's property.  It's a whole path....

Reply:
😕 OK